### ISSUE TITLE
Add button to close image even in SETM

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#5201]
Would be nice if there was a button to close the image in the Editor, even in Single Editor Tab Mode.
Reasons:
* Free up RAM for the queue.
* Write the PP3 to disk now and close the image to avoid accidentally altering it later.

**Comment by Thanatomanic** [comment]
Do we really need this? I use SETM exclusively and have never felt the need for a close button.

**Comment by Beep6581** [comment]
Yes, the only alternative is tediously restarting RawTherapee, and this is an improvement with no downsides.

**Comment by Thanatomanic** [comment]
If you are referring to accidentally altering an image, you can always go back in the history if you want to undo your edit before you open up another image.

Two more questions because I am skeptical:
1) What should you see when you close an image in SETM? Should the program send you back to the File Browser? 
2) When using the filmstrip it makes very little sense in terms of workflow to be able to close an image. I don't think any of our 'competitors' work like that too.

**Comment by Beep6581** [comment]
> If you are referring to accidentally altering an image, you can always go back in the history if you want to undo your edit before you open up another image.

Doing something accidentally sometimes implies not being aware of having done that thing. This feature takes that risk off the table.

> What should you see when you close an image in SETM? Should the program send you back to the File Browser?

It doesn't matter and I don't care.

> When using the filmstrip it makes very little sense in terms of workflow to be able to close an image.

It makes sense to me for the two reasons I listed in my first post.
Regarding RAM, see http://rawpedia.rawtherapee.com/The_Floating_Point_Engine#Memory_Requirements

> I don't think any of our 'competitors' work like that too.

They do what they need to given their circumstances. If having an image open means the user risks accidentally destroying his work and if having an image open means a potentially large amount of RAM is unavailable for things the user could be simultaneously doing in another tab such as processing queued shots, then I'm sure those competitors would implement the necessary features to handle that situation.

It's also odd that I'm forced to have an image open when I really don't want to, only because there is no way to close it other than restarting the program.

This is not a paradigm shift, just a simple "x" button which is already present in other modes.

**Comment by Benitoite** [comment]
I have to agree, being able to close out an image to finalize the .pp3 without having to reboot the software would be nice. 💈
